src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx

```diff
@@ -78,6 +78,7 @@ const VirtualMachineNavPageTitle: FC<VirtualMachineNavPageTitleProps> = ({ name,
         </Split>
       </span>
       <VirtualMachinePendingChangesAlert vm={vm} vmi={vmi} />
+      <StoppingAlert actions={actions} vmStatus={vm?.status?.printableStatus} />
```

Comment:
I think it will be better to get vm as a prop here and then in the component u can extract the status and also make onClick=()=>stopVM(vm, {gracePeriod: 0,}) and not filtering actions